packages/solana/src/utils/hash.ts

```diff
@@ -0,0 +1,42 @@
+import { sha256 } from '@ethersproject/sha2'
+import { Buffer } from 'buffer'
+import { PublicKey } from '@solana/web3.js'
+import { SolConstantsUtil } from '@web3modal/scaffold-utils/solana'
+
+export function getHashedName(name: string): Buffer {
+  const input = SolConstantsUtil.HASH_PREFIX + name
+  const str = sha256(Buffer.from(input, 'utf8')).slice(2)
+
+  return Buffer.from(str, 'hex')
+}
+
+if (typeof window !== 'undefined') {
+  // @ts-expect-error this
+  window.getHashedName = getHashedName
+}
+
+export async function getNameAccountKey(
+  hashed_name: Buffer,
+  nameClass?: PublicKey,
+  nameParent?: PublicKey
+): Promise<PublicKey> {
+  const seeds = [hashed_name]
```

Comment:
Let's separate the preperation-execution-conclusion parts of the function body with a new line. This will help us to keep them in a good pattern to make them more readable.

```suggestion
  const seeds = [hashed_name]
  
```